services/core-web/src/tests/mocks/dataMocks.js

```diff
@@ -1702,6 +1702,25 @@ export const INSPECTORS_HASH = {
   "51b3a499-a474-4d52-be99-5c5123d7501c": "BLAH",
 };
 
+export const NODS = {
```

Comment:
At the very end of the file, there's already a `NOTICES_OF_DEPARTURE` const representing the earlier form of this data.  Line `6847`